src/NavalFlags/NavalFlagImage.ts

```diff
@@ -0,0 +1,19 @@
+class NavalFlagImage {
+  public readonly character: string;
+  private readonly inlineSvg: string;
+  private readonly viewBoxWidth: number;
+  private readonly viewBoxHeight: number;
+
+  constructor(character: string, inlineSvg: string, viewBoxWidth: number, viewBoxHeight: number) {
+    this.character = character;
+    this.inlineSvg = inlineSvg;
```

Comment:
It might be worth moving this logic to a common InlineSvgImage class and having NavalFlagImage take a character and one of those.